docs/User/SWCI_APIs.md

```diff
@@ -38,7 +38,7 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
 |`listTasks()`|This method displays the list of tasks that are registered into the Swarm Learning network.| |
```

Comment:
For listNodes() - This method displays the list of Swarm nodes that have registered and are currently active. 
For each Swarm node, it displays the Node type, Host IP, Port, Container name, UUID, parent UUID and the last received ‘i-amalive’ Timestamp.
 
SWCI and ML nodes are not displayed.